.github/actions/upgrade-integration-test/action.yaml

```diff
@@ -0,0 +1,25 @@
+# Runs upgrade suite of integration tests
+# Requires as input IMG ENV variable for make target
```

Comment:
Do we need to have this as description, since it's already clearly visible in the action definition?